### ISSUE TITLE
vr issue and object loader 

### ISSUE THREAD

**Issue by EslamNemr** [aframevr/aframe#3883]
aframe master version just extend the view when entering vr mode in in the two screen

![1](https://user-images.githubusercontent.com/42042051/49128678-e3ea0f00-f2e5-11e8-915a-28c4f44aa265.jpeg)
![2](https://user-images.githubusercontent.com/42042051/49128679-e482a580-f2e5-11e8-8a15-b1fda2abce4a.jpeg)

and aframe 0.8.2 thres issue loading obj with material 
![4](https://user-images.githubusercontent.com/42042051/49128709-0b40dc00-f2e6-11e8-9f80-2d7d54657038.jpeg)

object and material on current aframe master loading correctly
![3](https://user-images.githubusercontent.com/42042051/49128722-18f66180-f2e6-11e8-9f68-aa32e1dc8e0c.jpeg)

**Comment by dmarcos** [comment]
What browser, device? Do you have a link to an example to reproduce?

**Comment by EslamNemr** [comment]
my glitch not working the code but https://glitch.com/edit/#!/join/187b6783-9ddc-4f98-8ef3-860cd0868e13
and for live http://www.eskanengineering.ae/VR/VR.html  
for live using last master
windows 10
browser chrome70.035
firefox 63.0.3
ie edge

and for mobile
iphone 7
ios 12 
safari and chrome

androied 5.1.1
firefox and chrome

**Comment by dmarcos** [comment]
What phone? The problem above with stereo rendering seems to happen in mobile. Do you see the problem in both iOS and Android? In browsers with WebVR enabled (Daydream)?

**Comment by EslamNemr** [comment]
the problem in ios and androied both
iphone 7 plus 
sony xperia z5 
but vr working well with aframe 0.8.0

webvr enabled

i attached browser photo for obj loader issue with 0.8.0 and 0.8.2

**Comment by EslamNemr** [comment]
for phone ios
chrome version 70.0.3538.75
safari 12.1


androied 
chrome 70.0.3538.110
firefox
63.0.2

**Comment by EslamNemr** [comment]
vr issue seems solved after remove the line i added
this.renderer.setPixelRatio(window.devicePixelRatio);   after line 75654
this line because  resolution dropped for picture and obj after exiting vr 
any other solution for this resolution dropped after exit VR ?

**Comment by EslamNemr** [comment]
yes same result given the same height and width in enter and exit vr event
but i debug on windows browser 
may be stereo rendering the reason in changing render size
i solve it with add this.renderer.setPixelRatio(window.devicePixelRatio); 
in exit vr event listener for now.

**Comment by dmarcos** [comment]
@EslamNemr can you supply a working link to reproduce?
